src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,116 @@
+@use "../../variables" as *;
+.footer {
+  .footer-top {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 52px;
+
+    .switch {
+      margin-bottom: 20px;
+      .switch-img {
+        width: 150px;
+      }
+    }
+
+    address {
+      text-align: center;
+      font-size: 14px;
+      line-height: 28px;
+      margin-bottom: 20px;
+    }
+
+    .icon {
+      display: flex;
+      gap: 14px;
+
+      a {
+        color: $black;
+        text-decoration: none;
+        transition: 0.2s ease;
+        &:hover {
+          color: $gray_nobel;
+        }
+      }
+    }
+  }
+  .link {
+    background-color: $white_smoke;
+    padding-left: 0;
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    padding: 16px;
+    border-top: 1px solid;
+
+    li {
+      list-style: none;
+      padding: 0 8px;
+      border-left: 1px solid;
+      line-height: 12px;
+
+      a {
+        text-decoration: none;
+        color: $black;
+        transition: 0.2s ease;
+        &:hover {
```

```suggestion
        text-decoration: none;
        color: $black;
        transition: 0.2s ease;
        font-size: $fontsize-s;
        &:hover {
```
lg 以上のときフォントサイズが少し大きいので、このように追加お願いします。